docs/developer-guide/rosplane/estimator/estimator-example.md

```diff
@@ -0,0 +1,373 @@
+    S^{-1} = (R_{accel} + CPC^\top)^{-1}
+\end{equation}
+
+This intermediate value is then used to find $L$:
+
+\begin{equation}
+    L = PC^\top S^{-1}
+\end{equation}
+
+The optimal estimate is then found:
+
+\begin{equation}
+    \hat{x}_a^+ =  \hat{x}_a^- + L (y - h)
+\end{equation}
+
+Finally, we update the covariance from our new estimate:
+
+\begin{equation}
+    P^+ = (I - LC) P^- (I - LC)^\top + LR_{accel}L^\top
+\end{equation}
+
+We repeat this cycle until termination of the program.
+This estimation scheme split into two parts allows for a clear set of equations and how the estimates affect one another.
+We will now move on to the next portion of the estimator that estimates the rest of the states.
+
+## Position Estimation
+
+The position estimation step follows the same algorithm as previous.
+Only new values are used for each of the matrices.
+Those new entries are shown here, but reference the previous section for details on implementation.
+
+### Propagation
+
+The attitude estimation is propagated according to the model $f$:
+
+\begin{equation}
+    f =
+    \begin{bmatrix}
+        V_g \cos{\chi} \\
+        V_g \sin{\chi} \\
+        \dot{V_g} \\
+        0 \\
+        0 \\
+        \dot{\psi} \\
+    \end{bmatrix}
+\end{equation}
+
+Where,
+
+\begin{equation}
+    \dot{\psi} = q \sin{\phi} + r \frac{\cos{\phi}}{\cos{\theta}} \\
+    \dot{V_g} = \frac{V_a}{V_g} \dot{\psi} (w_e \cos{\psi} - w_n \sin{\psi})
+\end{equation}
+
+This propagated estimate is then used in the calculation of the Jacobian $A$.
+
+\begin{equation}
+    A =
+    \begin{bmatrix}
+        0 & 0 & \cos{\chi} & -V_g \sin{\chi} & 0 & 0 & 0 \\
+        0 & 0 & \sin{\chi} & V_g \cos{\chi} & 0 & 0 & 0 \\
+        0 & 0 & -\frac{\dot{V_g}}{V_g} & 0 & -\dot{\psi} V_a \frac{\sin{\psi}}{V_g} & \dot{\psi} V_a \frac{\cos{\psi}}{V_g} & -\dot{\psi} V_a (w_n \cos{\psi} + w_e \sin{\psi}) \\
+        0 & 0 & -\frac{g}{V_g^2} \tan{\phi} & 0 & 0 & 0 & 0 \\
+        0 & 0 & 0 & 0 & 0 & 0 & 0 \\
+        0 & 0 & 0 & 0 & 0 & 0 & 0 \\
+        0 & 0 & 0 & 0 & 0 & 0 & 0 \\
+    \end{bmatrix}
+\end{equation}
+
+This Jacobian is then used to find a second-order approximation of the matrix exponential, $A_d$.
+
+\begin{equation}
+    A_d = I + \frac{T_s}{N} A + \frac{1}{2} \frac{T_s^2}{N^2} A^2
+\end{equation}
+
+Where $T_s$ is the length of a time step.
+$A_d$ is then used to propagate the actual covariance of the estimate.
+
+The process noise due to model uncertainty is defined in the matrix $Q$ and is fused into the covariance.
+This is done by the following equation:
+
+$$P_a = A_d P A_d^\top + Q \frac{T_s^2}{N^2}$$
+
+With this propagated estimate and covariance we are now ready for a measurement update.
+
+### Measurement Update
+
+<!-- TODO: update the documentation when the position step is refactored to happen all at once. -->
+
+Because the GPS measures come in slower than the model propagates, the measurement step is only run when their is new GPS information.
```

Comment:
their -> there

Reply:
I promise I know the difference! I don't know why I do this!!!!!!